scripts/get_version.py

```diff
@@ -0,0 +1,55 @@
+import requests
+from VSTSRequest import APIRequest
```

Comment:
wouldn't be suprised if vsts had a pip pacakge for dealign with their api.

Reply:
They have a Beta package that I know of. I can call the one REST API I need for now instead of importing their entire unstable package :)